app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -149,8 +149,9 @@ def find_host_by_provider_id(provider_id)
     def scheme_to_provider_mapping
       @scheme_to_provider_mapping ||= begin
         {
-          'gce' => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
-          'aws' => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'gce'   => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
+          'aws'   => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'azure' => ['ManageIQ::Providers::Azure::CloudManager'.safe_constantize, :uid_ems],
```

Comment:
Sorry about the last comment, that was bad. :-)

What are you trying to refer to here? :uid_ems in AWS seems to be blank and in OpenStack and Azure, it has a different meaning.

So maybe you want to use the name everywhere, or guid?